Sources/MessagingPush/MessagingPush.swift

```diff
@@ -66,7 +66,9 @@ public class MessagingPush: ModuleTopLevelObject<MessagingPushInstance>, Messagi
         let logger = diGraph.logger
         logger.debug("Setting up MessagingPush module...")
 
-        diGraph.automaticPushClickHandling.start()
+        if diGraph.sdkConfig.autoTrackPushEvents {
+            diGraph.automaticPushClickHandling.start()
+        }
```

Comment:
In a [previous commit in this PR](https://github.com/customerio/customerio-ios/pull/430/commits/f421379731d981565488c0d07b56c44a0f895346#diff-d322168af6d12b45ff1a57021d6a686396fe280ff72e00f2c1872bc9e4abff3fR69-R71) I added a new messaging push module sdk config option to enable/disable this feature. 

However, I later realized that our Sdk already comes with a config option that makes sense to re-use. I want to avoid confusing customers by having 2 config options in the SDK that have very similar names and use cases.